web_src/js/components/DashboardRepoList.vue

```diff
@@ -354,7 +359,12 @@ export default defineComponent({
           <svg-icon name="octicon-plus"/>
         </a>
       </h4>
-      <div class="ui attached segment repos-search">
+      <div v-if="isLoading && !reposTotalCount" class="ui attached segment" :class="{'is-loading': isLoading}"/>
+      <div v-if="isRepoEmpty" class="ui attached segment empty-placeholder">
```

Comment:
`isRepoEmpty` already mixes the `isLoading` and `reposTotalCount` logic, now here: these 3 variables appear at the same place, more unclear .....

I think `isRepoEmpty` should be removed.